src/main/java/me/botsko/prism/actions/ActionMeter.java

```diff
@@ -13,31 +14,49 @@
  */
 public class ActionMeter {
     private static final Map<String, DripMeter> meter = new HashMap<>();
-    private static  boolean monitoring = false;
+    private static boolean monitoring = false;
+
+    /**
+     * Returns a map of Metrics for the actions.
+     *
+     * @return Map
+     */
+    public static Map<String, Integer> getMetricMeter() {
+        Map<String, Integer> out = new HashMap<>(metricMeter);
+        metricMeter.clear();
+        return ImmutableMap.copyOf(out);
+    }
+
+    private static final Map<String, Integer> metricMeter = new HashMap<>();
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6279951)